### PR TITLE
FIX(mac, audio): Identify output device

### PR DESCRIPTION
**Pull request by TerryGeng**
Some USB audio interfaces have both input and output stream,
but they are accidentally ignored in #4694.

Independent from that this PR changes the macro names for saving
and loading values to and from the settings in order to make that part
of the code less confusing.

### CODE REVIEW

**Comment by Krzmbrzl** [comment]
@TerryGeng I think the second commit is something that I already fixed

**Comment by TerryGeng** [comment]
@Krzmbrzl Yeah you are right... I just rebased my changes and the second commit just disappeared and confused me for a while.

**Comment by TerryGeng** [comment]
BTW I think redefining `SAVELOAD` in the middle of `Settings.cpp` is... very confusing.

**Comment by Krzmbrzl** [comment]
I agree. I think the very name _SAVELOAD_ is very confusing to begin with. I assume this was done so that the code for saving a value can be copy&pasted to the part that loads it... :shrug:

**Comment by TerryGeng** [comment]
Make sense. But now the code has grown larger and subtle differences for loading specific items emerge, simply copying and pasting won't work anymore.

I will just separate `SAVE` and `LOAD`.

File: src/mumble/Settings.cpp [hunk 1]
```diff
@@ -1017,8 +1017,7 @@ void Settings::load(QSettings *settings_ptr) {
 	settings_ptr->endGroup();
 }
 
-#undef SAVELOAD
-#define SAVELOAD(var, name)                               \
+#define SAVE(var, name)                               \
```
Comment (Krzmbrzl) [inline src/mumble/Settings.cpp:1020]:
```suggestion
#undef LOAD
#define SAVE(var, name)                               \
```
This way you can't accidentally load in the `save` function (aka: prevents copy&paste errors)

**Review by Krzmbrzl — CHANGES_REQUESTED** [review]

File: src/mumble/Settings.cpp (hunk 1 above)
Comment (TerryGeng) [reply src/mumble/Settings.cpp:1020]: Very smart :)

**Review by Krzmbrzl — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
Thanks for the follow-up :+1:

**Pull request merged**